### PR TITLE
Resolving 'Count only one vote per person per message.'

### LINKED ISSUE #9: Count only one vote per person per message.
A person should be only able to vote once on each message.

### PR DESCRIPTION
**Pull request by joao-v-maia**
Creating this to resolve #9 with already_voted() function. Maybe this should be treated as a database issue, but we can discuss that later <3

### CODE REVIEW

File: karmagrambot/__init__.py [hunk 1]
```diff
@@ -8,13 +8,16 @@
 
 logging.basicConfig()
 
+def already_voted(replied, user_id):
```
Comment (tarcisioe) [inline karmagrambot/__init__.py]: Please add a docstring to new functions.

Let's adopt Google style for docstrings:

```
<Short description>

Args:
    arg_name: <description>

Returns:
    <description>
```

Comment (tarcisioe) [inline karmagrambot/__init__.py]: Let's also add static typing as we go:

```
def already_voted(replied: str, user_id: str) -> bool:
```

**Review by tarcisioe — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: karmagrambot/__init__.py (hunk 1 above)
Comment (joao-v-maia) [reply karmagrambot/__init__.py]: Done

Comment (joao-v-maia) [reply karmagrambot/__init__.py]: Done

**Review by tarcisioe — APPROVED** [review]

**Pull request merged**